static/js/mediastream/api.js

```diff
@@ -32,13 +32,12 @@ define(['jquery', 'underscore', 'ua-parser'], function($, _, uaparser) {
 
 		this.e = $({});
 
-		var ua = uaparser();
-		if (ua.os.name && /Spreed Desktop Caller/i.test(ua.ua)) {
-			this.userAgent = ua.ua.match(/Spreed Desktop Caller\/([\d.]+)/i)[1] + " (" + ua.os.name + ")";
-		} else if (ua.browser.name) {
-			this.userAgent = ua.browser.name + " " + ua.browser.major;
+		if ($.ua.os.name && /Spreed Desktop Caller/i.test($.ua.ua)) {
```

Comment:
Why did you change this, uaparser should not be used from jquery - it should be used as AMD module.